### ISSUE TITLE
BUG: FileNotFoundError

### ISSUE THREAD

**Issue by Maarten-vd-Sande** [vanheeringen-lab/snakemake-workflows#309]
Last night I got this error, and it doesn't matter whether or not I execute the workflow on `moldev` or locally.

```
Traceback (most recent call last):
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/__init__.py", line 647, in snakemake
    keepincomplete=keep_incomplete,
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/workflow.py", line 894, in execute
    success = scheduler.schedule()
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/scheduler.py", line 360, in schedule
    run = self.job_selector(needrun)
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/scheduler.py", line 504, in job_selector
    c = list(map(self.job_reward, jobs))  # job rewards
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/scheduler.py", line 588, in job_reward
    temp_size = self.dag.temp_size(job)
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/dag.py", line 548, in temp_size
    return sum(f.size for f in self.temp_input(job))
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/dag.py", line 548, in <genexpr>
    return sum(f.size for f in self.temp_input(job))
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/io.py", line 139, in wrapper
    return func(self, *args, **kwargs)
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/io.py", line 154, in wrapper
    return func(self, *args, **kwargs)
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/io.py", line 350, in size
    return self.size_local
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/io.py", line 355, in size_local
    self.check_broken_symlink()
  File "/home/sande/anaconda3/envs/snakemake-workflows/lib/python3.6/site-packages/snakemake/io.py", line 360, in check_broken_symlink
    if not self.exists_local and os.lstat(self.file):
: [Errno 2] No such file or directory: '/mnt/ocimum/vol/new_atac/fastq_trimmed/Xt_stage34_rep2_R1_trimmed.fastq.gz'
```

I am not sure if this is a problem caused by us, or a problem of SnakeMake.

**Comment by Maarten-vd-Sande** [comment]
It is really weird, because I think the rule is just never executed and Snakemake expects its output... On the slurm.out file it just prints the config and then waits for output, while on other rules it also prints the Dag etc.. 

I'm relaly confused by this

**Comment by siebrenf** [comment]
this looks like the issue I had with the blacklist (which you managed to fix 👍 ). Is the error maybe occurring inside a rule, and is the file obtained from the params instead of the input?

**Comment by Maarten-vd-Sande** [comment]
No I found it, but it is because merge replicates removes the original file and fastqc wants to use those files and doesn't realize they are gone.

**Comment by Maarten-vd-Sande** [comment]
There are quite some small issues atm with running it on a big dataset, I started some fixes but there are still isues:
https://github.com/vanheeringen-lab/snakemake-workflows/tree/small_patches

**Comment by Maarten-vd-Sande** [comment]
Another issue is that deeptools fills the `tmp` folder with lots of files until your computer is full....

**Comment by Maarten-vd-Sande** [comment]
Another issue is that somewhere the .fa.sizes is emptied :thinking:

**Comment by Maarten-vd-Sande** [comment]
Solved in #310
